grid.cpp

```diff
@@ -192,6 +208,34 @@ void initializeGrids(
          exit(1);
       }
       phiprof::stop("Read restart");
+
+      // For now, alpha is read from the restart file
+      phiprof::start("Re-refine spatial cells");
+      for (int i = 0; P::adaptRefinement && i < P::amrMaxSpatialRefLevel; ++i) {
```

Comment:
Having a boolean check inside the for-loop condition is technically correct but kinda odd to me :)